CountryPicker/CountryPicker/Country.swift

```diff
@@ -12,13 +12,13 @@ import UIKit
 open class Country {
 
     // MARK:- Variable
-    open var countryCode: String
+    open private(set) var countryCode: String
```

Comment:
why restricted the access of `countryCode` ?

Reply:
If `countryCode` has a public setter then the name, code & dialing code could be changed independently and be out of sync :)